src/pages/UpdateShelter/UpdateShelter.tsx

```diff
@@ -209,15 +213,51 @@ const UpdateShelter = () => {
               error={!!errors.shelteredPeople}
               helperText={errors.shelteredPeople}
             />
-            <SelectField
-              label="O abrigo aceita animais"
-              value={values.petFriendly ? 'true' : 'false'}
-              onSelectChange={(v) => setFieldValue('petFriendly', v === 'true')}
-              options={[
-                { value: 'true', label: 'Sim' },
-                { value: 'false', label: 'Não' },
-              ]}
-            />
+
+            <Card className='w-full'>
+              <CardHeader className='flex flex-row items-center gap-1 w-full'>
+              <PawPrint />
+                <CardTitle>
+                  Pets
+                </CardTitle>
+              </CardHeader>
+              <CardContent className='space-y-4'>
+                <SelectField
+                  label="O abrigo aceita animais"
+                  value={values.petFriendly ? 'true' : 'false'}
+                  onSelectChange={(v) => {
+                    setFieldValue('petFriendly', v === 'true')
+                    setFieldValue('petsCapacity', v === 'true' ? values.petsCapacity : 0)
+                    setFieldValue('shelteredPets', v === 'true' ? values.shelteredPets : 0)
+                  } }
```

Comment:
E para as demais ficaria da seguinte forma:

```suggestion
                    setFieldValue('petsCapacity', v ? values.petsCapacity : 0)
                    setFieldValue('shelteredPets', v ? values.shelteredPets : 0)
                  } }
```

Ficaria mais simples a leitura